lib/curve25519.d.ts

```diff
@@ -0,0 +1,14 @@
+export namespace curve25519 {
+    export { sign };
+    export { verify };
+    export { keygen };
+}
+/********* DIGITAL SIGNATURES *********/
+declare function sign(h: any, x: any, s: any): any[] | undefined;

```

Comment:
Can we have a better type on any of these? I know this is an outside package with no types, but at least the return types should be stronger, otherwise

```
const value = curve.sign(...) // value is any which means no typecheck from there onwards
```

Reply:
Yes, this file was generated for a 3rd party js library file that did not have any type definition and I did not modify it.

I will try to improve the types here before merge.